crates/ruff_linter/src/rules/flake8_use_pathlib/snapshots/ruff_linter__rules__flake8_use_pathlib__tests__full_name.py.snap

```diff
@@ -317,3 +317,33 @@ full_name.py:47:1: PTH123 `open()` should be replaced by `Path.open()`
    | ^^^^ PTH123
 48 | open(p, 'r', - 1, None, None, None, False, opener)
    |
+
+full_name.py:65:1: PTH123 `open()` should be replaced by `Path.open()`
```

Comment:
I haven't looked at the issue more closely, but does the handling of bytes string is planned as a follow-up or it's to be avoided for now. Given that the tests are not removed, I'm guessing the former. If so, it'd be useful to add a TODO (or possibly remove the tests.)

Reply:
We want to keep flagging them. So an exmaple seems fine.